README.md

```diff
@@ -48,6 +51,30 @@ planx-new is a monorepo containing our full application stack. Here's a quick su
 
 9. Open `http://localhost:3000` and login with your Google email address
 
+**If you're not a member of OSL, follow these steps:**
+
+1. Fork or clone this repository
+
+2. Copy all `.example` env files into their respective directories and replace 👻 with a string like "SECRET" or longer where noted
+
+3. Setup free OAuth Client ID credentails in the [Google Cloud APIs console](https://console.cloud.google.com/apis/credentials)
+  - Application type = "Web application"
+  - Authorised JavaScript origins = "http://localhost:3000"
+  - Authorised redirect URIs = "http://localhost:7002/auth/google/callback"
+  - Update the `GOOGLE_CLIENT_ID` & `GOOGLE_CLIENT_SECRET` env vars with your new credentials
+
+4. Run `pnpm start` from the project root to set up docker containers for the application's backend (postgres, sharedb, api and hasura server processes). Please note you will not be able to run commands that sync seed data from production.
+
+5. Move into the hasura directory `cd ../hasura.planx.uk` and install dependencies `pnpm i`.
+
+6. Open [Hasura's](https://hasura.io/) web console (`cd hasura.planx.uk` then `pnpm start`) and add your Google email address to the `users` table. You'll also likely want to create an initial `team`. This will eventually allow you to authenticate into the application as an admin.
+
+7. Follow steps 7-9 above to start the editor and login !
+
+At this point you'll be running the full Planx application locally in a docker container. See our Github Actions pull request workflow as an example of how to deploy via docker to a virtual linux server, or explore the `infrastructure` directory for how to deploy via Pulumi infrastructure-as-code to AWS. 
+
+We'd love to hear what you're building on Planx, don't hesitate to get in touch with questions.
```

Comment:
_suggestion_: it might be nice to point at a preferred contact method if we have one?